clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBStoragePartition.java

```diff
@@ -163,10 +165,10 @@ protected RocksDBStoragePartition(
     super(storagePartitionConfig.getPartitionId());
     this.factory = factory;
     this.rocksDBServerConfig = rocksDBServerConfig;
-    // Create the folder for storage partition if it doesn't exist
-    this.storeName = storagePartitionConfig.getStoreName();
-    this.storeNameWithoutVersionSuffix = Version.parseStoreFromVersionTopic(storeName);
+    this.versionedStoreName = storagePartitionConfig.getStoreName();
+    this.storeName = Version.parseStoreFromVersionTopic(versionedStoreName);
     this.partitionId = storagePartitionConfig.getPartitionId();
+    this.replicaId = Utils.getReplicaId(storagePartitionConfig.getStoreName(), partitionId);
```

Comment:
nit: could just `versionedStoreName` and I personally prefer `storeNameWithVersion` over `versionedStoreName` but it's okay to drop this comment.

Reply:
Updated